core/txdb/internal/storage/snapshot.proto

```diff
@@ -9,11 +9,11 @@ message Snapshot {
   // The nodes are ordered according to a pre-order traversal.
   repeated StateTreeNode nodes = 1;
 
-  // Issuances contains the record of recent issuances for ensuring uniqueness
-  // of issuances.
-  repeated Issuance issuances = 2;
```

Comment:
Even if it's not philosophically orthogonal, it's definitely architecturally and practically orthogonal. I don't think there should be a single atomic PR implementing the spec changes.